ompi/mca/common/ompio/Makefile.am

```diff
@@ -30,7 +30,11 @@ sources = \
 	common_ompio_file_view.c   \
 	common_ompio_file_read.c   \
 	common_ompio_file_write.c
+else
 
+headers = 
+soruces = 
```

Comment:
@edgargabriel there is a minor typo, it should be ```sources=```

fwiw, i made things a bit differently in #2018